### ISSUE TITLE
Update compiler flags on OSX Catalina

### ISSUE THREAD

**Issue by phargogh** [natcap/invest#98]
This came up very recently when starting to run tests on Mac OS X (see #96): OS X Catalina changed the name of the C++ standard library and so compiling from source yields warnings like this:

```
warning: include path for stdlibc++ headers not found; pass '-stdlib=libc++' on the command line to use the libc++ standard library instead [-Wstdlibcxx-not-found]
```

We fixed this for pygeoprocessing over in https://github.com/natcap/pygeoprocessing/issues/9, and so a similar approach should work here.

**Comment by phargogh** [comment]
I'm working on this in my fork on `bugfix/98-update-compiler-flags-osx-catalina`.

**Comment by davemfish** [comment]
resolved in #110
